autoPyTorch/datasets/resampling_strategy.py

```diff
@@ -39,13 +39,6 @@ def __call__(self, random_state: np.random.RandomState, val_share: float,
         ...
 
 
-class NoResamplingFunc(Protocol):
-    def __call__(self,
-                 random_state: np.random.RandomState,
-                 indices: np.ndarray) -> np.ndarray:
-        ...
```

Comment:
What this change is supposed to mean? (I am lost)

Reply:
these are removing some artifacts that came due to rebasing from development

Reply:
It was added twice.